src/main/java/org/kaiaccount/account/eco/commands/bank/DeletePlayerBankAccountCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.kaiaccount.account.eco.commands.bank;
+
+import org.bukkit.OfflinePlayer;
+import org.jetbrains.annotations.NotNull;
+import org.kaiaccount.AccountInterface;
+import org.kaiaccount.account.eco.message.Messages;
+import org.kaiaccount.account.eco.message.messages.error.SourceOnlyCommandMessage;
+import org.kaiaccount.account.inter.type.named.bank.player.PlayerBankAccount;
+import org.kaiaccount.account.inter.type.player.PlayerAccount;
+import org.mose.command.ArgumentCommand;
+import org.mose.command.CommandArgument;
+import org.mose.command.arguments.operation.ExactArgument;
+import org.mose.command.arguments.simple.text.StringArgument;
+import org.mose.command.context.CommandContext;
+
+import java.util.List;
+import java.util.Optional;
+
+public class DeletePlayerBankAccountCommand implements ArgumentCommand {
+
+    public static final ExactArgument CREATE = new ExactArgument("close");
+    public static final StringArgument NAME = new StringArgument("name");
+
+    @Override
+    public @NotNull List<CommandArgument<?>> getArguments() {
+        return List.of(CREATE, NAME);
+    }
+
+    @Override
+    public @NotNull String getDescription() {
+        return "Deletes a bank account";
+    }
+
+    @Override
+    public @NotNull Optional<String> getPermissionNode() {
+        return Optional.empty();
```

Comment:
I would have it check the player create bank self permission, this is so if the player cant create a bank, they cant delete a bank and therefore wont be offered the command in suggestion. 

If no permission is given, the command system will assume that the command is valid and offer it as suggestions